packages/iTwinUI-react/README.md

```diff
@@ -0,0 +1,99 @@
+<p align="center">
```

Comment:
hmm.. I would say this still should be in our main readme so users firstly opening github, sees all info.. they dont really care about our monorepo structure or info about it..

Reply:
I kept this README here for our storybook Overview page and NPM package. I think we need something different for the main README but I agree current info is not enough as it only talks about monorepo

Reply:
Well, i'd say better keep our old readme than showing this. And then we can see, what we want to improve there.
Cause honestly, we still deliver only itwinui-react from here, everything else is applicable only to contributors. So our repo structure should not influence that.